.github/pull_request_template.md

```diff
@@ -2,6 +2,10 @@
 
 Write a brief explainer on your code changes.
 
+## What is the related Linear task?
+
+Link to linear ticket.
+
 ## Please explain the objectives of your changes below
 
 Put down any required details on the broader aspect of your changes. If there are any dependent changes, **mandatorily** mention them here
```

Comment:
There's too much of content under `Type of change`. Remove all that & convert to question in bold:
### Any changes to existing capabilities/behaviour, mention the reason & what are the changes ?